test/unittests/Loris_PHPUnit_Database_TestCase.php

```diff
@@ -89,6 +89,7 @@ protected function setUp()
      *
      * @return \PHPUnit_Extensions_Database_DB_DefaultDatabaseConnection
      */
+    /* @phan-suppress-next-line PhanUndeclaredTypeReturnType*/
```

Comment:
Why is this here? It says above it the return type is PHPUnit_Extensions_Database_DB_DefaultDatabaseConnection

Reply:
I'm not sure, Phan seemed to have an issue with it at one stage. I'll go back and verify that this is the case.

Reply:
This issue is fixed in #4274 which is based on this PR. If you merge this one first then this line will be removed and the bug fixed.